app/styles/style.css

```diff
@@ -131,4 +137,270 @@ body {
     text-decoration: none;
     display: flex;
   }
+
+  .jobs {
+    padding: 24px;
+  }
```

Comment:
[OPTIONAL] have you considered using a grid for the jobs section? It could help make things look more organized and visually appealing.😄